src/forms/helpers/dropdown-select.js

```diff
@@ -22,12 +24,13 @@ function DropdownSelect ({
   className,
   expanded, 
   selectedValues, 
-  toggleExpanded, 
+  toggleExpanded,
+  selectedOptionsDisplayFormatter,
 }) {
   return (
     <div className="dropdown-select">
       <div className="select-input" onClick={ toggleExpanded }>
-        <p>{ getLabel(selectedValues) }</p>
+        <p>{selectedOptionsDisplayFormatter(selectedValues)}</p>
```

Comment:
Since this has the possibility to return jsx, should we make it a component (a la [LabeledField's labelComponent](https://github.com/LaunchPadLab/lp-components/blob/adc4113fef8b1b9ca115daee6632b5de70ab2717/src/forms/labels/labeled-field.js#L87))?

Reply:
I typically would interpret a "Formatter" as something that returns a string, but I might just be biased by my experience with date libraries (e.g., [date-fns](https://date-fns.org/v2.26.0/docs/format) and [moment](https://momentjs.com/docs/#/displaying/format/))